.github/workflows/ci.yml

```
@@ -0,0 +1,24 @@
+
+name: Rust
+
+on:
+  push:
+    branches: [ "master" ]
```

YAMLlint (1.35.1)

[error] too many spaces inside brackets

(brackets)